### ISSUE TITLE
Turn `AcceleratedVideo` example into a documentation page

### ISSUE THREAD

**Issue by JonnyBurger** [remotion-dev/remotion#1406]
I just wrote the following Discord message about how to accelerate the speed of a video over time:

> hey @benk#3758! this is a good idea in theory, the reason it doesn't work though is that this violates a remotion rule that requires each frame having to render the same in isolation regardless of which frames came before it or come after it.

>let's say we render frame `100`, the component would render and it would determine the playbackRate to be `2`, but remotion doesn't realize that the playbackrate was interpolated from 1 to 2. therefore it renders as it the whole video was played at 2x speed, leading to a choppy video.

> to make the code declarative, we have to count together how much of the video has passed by adding together all the playback rates we had of the previous frames, and start the video at this position, and then we can pass the same playback rate. a bit hard to wrap your head around, but here is a code snippet that makes your intention work!

```tsx
import React from 'react';
import {
    AbsoluteFill,
    interpolate,
    Sequence,
    useCurrentFrame,
    Video,
} from 'remotion';

const remapSpeed = ({
    frame,
    speed,
}: {
    frame: number;
    speed: (fr: number) => number;
}) => {
    let framesPassed = 0;
    for (let i = 0; i <= frame; i++) {
        framesPassed += speed(i);
    }

    return framesPassed;
};

export const AcceleratedVideo: React.FC = () => {
    const frame = useCurrentFrame();
    const speedFunction = (f: number) =>
        interpolate(f, [0, 500], [1, 5], {
            extrapolateLeft: 'clamp',
            extrapolateRight: 'clamp',
        });

    const remappedFrame = remapSpeed({
        frame,
        speed: speedFunction,
    });

    return (
        <AbsoluteFill>
            <Sequence from={frame}>
                <Video
                    startFrom={Math.round(remappedFrame)}
                    playbackRate={speedFunction(frame)}
                    src="https://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4"
                />
            </Sequence>
        </AbsoluteFill>
    );
};
```

It would be cool if we added this to the Snippets section (with better wording possibly than just some chat server message explanation) and added an embedded player (same as it happened in #1392)

**Comment by JonnyBurger** [comment]
@kaf-lamed-beyt Happily!

**Comment by kaf-lamed-beyt** [comment]
Great! 💥🚀

**Comment by kaf-lamed-beyt** [comment]
Quick one @JonnyBurger,

Can I get the link to the question asked by @BenK#3758 in the discord server?

**Comment by kaf-lamed-beyt** [comment]
I'm still getting these errors after I've done `pnpm i` to get the new deps

![Screenshot from 2022-10-31 09-16-52](https://user-images.githubusercontent.com/48127194/198964902-ff3fe531-ad60-4efa-a328-4e9450228c2a.png)

![Screenshot from 2022-10-31 09-17-34](https://user-images.githubusercontent.com/48127194/198964936-7870d4a4-fc35-438d-b84a-c145503cbaf9.png)

**Comment by JonnyBurger** [comment]
@kaf-lamed-beyt The original question was just: `Could you please share in code how you accelerate the playback of a video with spring animation?`

I think you get the error because you did not run `pnpm build` in the root directory!

**Comment by kaf-lamed-beyt** [comment]
Thanks for pointing this out! 😎

**Comment by kaf-lamed-beyt** [comment]
This is what I'm getting now @JonnyBurger 
![Screenshot from 2022-10-31 18-18-16](https://user-images.githubusercontent.com/48127194/199069804-db0f0736-538d-43a8-869a-e87b0a5b03a2.png)

**Comment by JonnyBurger** [comment]
@kaf-lamed-beyt you need to run `npm start` from the docs package

**Comment by kaf-lamed-beyt** [comment]
Thank you for this @JonnyBurger.

I'm currently trying to add a sample to the doc. But, I don't know how to go about that.

**Comment by kaf-lamed-beyt** [comment]
Okay... 

I think I'm beginning to get a hang of how to go about it, by taking a look at this [diff](https://github.com/remotion-dev/remotion/pull/1392/files#diff-4b55dd267dd443c9cee373a2cf3a460e21d2ebbb31a4888532a4a3ccc83fb775)

I'll let you know when I encounter any issues, moving forward!

> Thank you for this @JonnyBurger.
> 
> I'm currently trying to add a sample to the doc. But, I don't know how to go about that.

**Comment by JonnyBurger** [comment]
Fixed by #1457
